src/components/QueryOutput/QueryOutput.js

```diff
@@ -1371,34 +1398,32 @@ export class QueryOutput extends React.Component {
     })
 
     queryFilters.forEach((queryFilter) => {
-      if (!allFilters.find((filter) => filter.name === queryFilter.name)) {
+      const byName = queryFilter?.name && tableFilterNames.has(queryFilter.name)
+      const byId = queryFilter?.id && tableFilterIds.has(queryFilter.id)
+      const existsInTableFilters = byName || byId
+
+      if (existsInTableFilters && !allFilters.find((filter) => filter.name === queryFilter.name)) {
```

Comment:
Why only push the filter if it already exists in table filters? Wouldnt we want to push the new filter anyway? This might break filter drilldowns (list drilldowns from charts) since we send in the drilldown as a fe_req filter not a table filter